internal/models/policy/recipe/mutation/pod_security.go

```diff
@@ -0,0 +1,512 @@
+type VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityPrivileged struct {
+
+	// Condition specifies whether to always mutate/set this value or only if container security context contains or does not contain this field
+	// Required: true
+	// Enum: [Always IfFieldExists IfFieldDoesNotExist]
+	Condition *string `json:"condition"`
+
+	// Value to set for privileged field in container security context
+	// Required: true
+	Value *bool `json:"value"`
+}
+
+// MarshalBinary interface implementation.
+func (m *VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityPrivileged) MarshalBinary() ([]byte, error) {
+	if m == nil {
+		return nil, nil
+	}
+
+	return swag.WriteJSON(m)
+}
+
+// UnmarshalBinary interface implementation.
+func (m *VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityPrivileged) UnmarshalBinary(b []byte) error {
+	var res VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityPrivileged
+	if err := swag.ReadJSON(b, &res); err != nil {
+		return err
+	}
+
+	*m = res
+
+	return nil
+}
+
+// VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityReadOnlyRootFilesystem Set readOnlyRootFilesystem flag in container security context.
+//
+// swagger:model VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityReadOnlyRootFilesystem
+type VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityReadOnlyRootFilesystem struct {
+
+	// Condition specifies whether to always mutate/set this value or only if container security context contains or does not contain this field
+	// Required: true
+	// Enum: [Always IfFieldExists IfFieldDoesNotExist]
+	Condition *string `json:"condition"`
+
+	// Value to set for readOnlyRootFilesystem field in container security context
+	// Required: true
+	Value *bool `json:"value"`
+}
+
+// MarshalBinary interface implementation.
+func (m *VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityReadOnlyRootFilesystem) MarshalBinary() ([]byte, error) {
+	if m == nil {
+		return nil, nil
+	}
+
+	return swag.WriteJSON(m)
+}
+
+// UnmarshalBinary interface implementation.
+func (m *VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityReadOnlyRootFilesystem) UnmarshalBinary(b []byte) error {
+	var res VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityReadOnlyRootFilesystem
+	if err := swag.ReadJSON(b, &res); err != nil {
+		return err
+	}
+
+	*m = res
+
+	return nil
+}
+
+// VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityRunAsGroup Set numerical group ID in runAsGroup flag in container security context.
+//
+// swagger:model VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityRunAsGroup
+type VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityRunAsGroup struct {
+
+	// Condition specifies whether to always mutate/set this value or only if container security context contains or does not contain this field
+	// Required: true
+	// Enum: [Always IfFieldExists IfFieldDoesNotExist]
```

Comment:
Isn't the enum is defined in the models?

Reply:
Following blocks were generated 11 times with same values, fields were different.


```
const (

	// VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityFsGroupConditionAlways captures enum value "Always"
	VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityFsGroupConditionAlways string = "Always"

	// VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityFsGroupConditionIfFieldExists captures enum value "IfFieldExists"
	VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityFsGroupConditionIfFieldExists string = "IfFieldExists"

	// VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityFsGroupConditionIfFieldDoesNotExist captures enum value "IfFieldDoesNotExist"
	VmwareTanzuManageV1alpha1CommonPolicySpecMutationV1PodSecurityFsGroupConditionIfFieldDoesNotExist string = "IfFieldDoesNotExist"
)
```
Meaning there were lot of duplication. Decided not to use it because schema validations are implemented. Let me know what approach you prefer or need.